### PR TITLE
Upgrading Ruby to 2.6

### LINKED ISSUE #1311: Update the ruby on the lib jobs server
Add `ruby_version_override: "ruby2.6"` to the vars and rerun the playbook like: https://github.com/pulibrary/princeton_ansible/blob/572625f009daa8256b3226f66bf0e98ea67964f7/group_vars/approvals/approvals_staging.yml#L21

### PR DESCRIPTION
**Pull request by carolyncole**
Adding variables to get the rails app up and running

fixes #1311

### CODE REVIEW

File: roles/pulibrary.lib-jobs/vars/main.yml [hunk 1]
```diff
@@ -19,8 +19,15 @@ rails_app_vars:
     value: "http://www.worldcat.org/webservices/catalog/content"
   - name: WORLDCAT_API_KEY
     value: '{{ vault_wc_key | default("key") }}'
+  - name: APP_SECRET_KEY_BASE
+    value: '{{ secret_key }}'
```
Comment (kayiwa) [inline roles/pulibrary.lib-jobs/vars/main.yml:23]: this fails (I rebased against master) with an `undefined variable for secret_key`

Comment (kayiwa) [reply roles/pulibrary.lib-jobs/vars/main.yml:23]: ping @carolyncole sorry this sat for a while.

**Comment by carolyncole** [comment]
@kayiwa sorry this took me a while to fix up...  The vault password changed mid PR, which made it hard to fix.  It should be good to go now...

**Comment by carolyncole** [comment]
All these changes seem to already be on the server.  I'm closing this...

**Pull request closed without merging**